<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>KineForce</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="4" column="2">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tabFormulaire">
       <attribute name="title">
        <string>Informations</string>
       </attribute>
       <layout class="QFormLayout" name="formLayout_2">
        <item row="0" column="0" colspan="2">
         <widget class="QGroupBox" name="groupeGB">
          <property name="title">
           <string>Groupe</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="0" colspan="2">
            <widget class="QLineEdit" name="groupSaveFileName"/>
           </item>
           <item row="0" column="2">
            <widget class="QPushButton" name="openStudyButton">
             <property name="text">
              <string>Sélectionner le fichier de sauvegarde du groupe</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QCheckBox" name="studyFormatCB">
             <property name="text">
              <string>Nouveau Fichier</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QSpinBox" name="rowSB"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Ligne de sauvegarde du participant dans le excel de groupe :</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QGroupBox" name="participantGB">
          <property name="title">
           <string>Participant</string>
          </property>
          <layout class="QFormLayout" name="formLayout">
           <item row="0" column="0">
            <widget class="QLineEdit" name="subjectnameEdit"/>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="nameLabel">
             <property name="text">
              <string>Identifiant du participant</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="footLengthLabel">
             <property name="text">
              <string>Longueur de pied du participant (cm)</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLabel" name="footWidthLabel">
             <property name="text">
              <string>Largeur de pied du participant (cm)</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QDoubleSpinBox" name="footWidthDSB"/>
           </item>
           <item row="1" column="0">
            <widget class="QDoubleSpinBox" name="footLengthDSB"/>
           </item>
          </layout>
         </widget>
        </item>
        <item row="3" column="0" colspan="2">
         <widget class="QGroupBox" name="essaiGB">
          <property name="title">
           <string>Fichier de données de l'essai</string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="3" column="1">
            <widget class="QPushButton" name="browse">
             <property name="text">
              <string>Ouvrir</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLineEdit" name="filename"/>
           </item>
          </layout>
         </widget>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="QGroupBox" name="SaveGB">
          <property name="title">
           <string>Emplacements de sauvegarde individuels</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="2" column="1">
            <widget class="QCheckBox" name="newsave">
             <property name="text">
              <string>Nouveau Fichier</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QPushButton" name="saveLocation">
             <property name="text">
              <string>Ouvrir</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLineEdit" name="savefilename"/>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Résultats (Par axe)</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabData">
       <attribute name="title">
        <string>Données</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="4" column="0">
         <widget class="QPushButton" name="BSPhaseStart">
          <property name="text">
           <string>Début de la phase BS</string>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QPushButton" name="Phase1Start">
          <property name="text">
           <string>Fin de l'atteinte</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QPushButton" name="BSPeak">
          <property name="text">
           <string>Sommet du BS</string>
          </property>
         </widget>
        </item>
        <item row="4" column="3">
         <widget class="QPushButton" name="maxBtn">
          <property name="text">
           <string>Max de la phase 1</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0" colspan="4">
         <widget class="QPushButton" name="save">
          <property name="text">
           <string>Calculer et Sauvegarder</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" rowspan="4" colspan="4">
         <widget class="QGroupBox" name="displayGB">
          <property name="title">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
